docs/topics/db/transactions.txt

```diff
@@ -232,6 +238,17 @@ Django provides a single API to control database transactions.
     is especially important if you're using :func:`atomic` in long-running
     processes, outside of Django's request / response cycle.
 
+.. warning:: Durability and testing
+
+    :class:`django.test.TestCase` disables the durability check to allow
```

Comment:
As a pytest user it makes me wonder whether I can use this. Should we also allow durability checking to be disabled via a setting?

Reply:
I think it should be possible for pytest-django to implement something similar.

Reply:
It should be 4 lines in a new or existing pytest fixture that uses try / finally to disable then re-enable the flag.

Reply:
Also I still prefer and recommend sticking with Django's `TestCase` when using pytest, it has many features not reproduced by pytest-django